assets/CSS/desktop.css

```diff
@@ -0,0 +1,680 @@
+    font-size: 1.6rem;
+    max-width: 144rem;
+  }
+
+  .flex-conatiner,
+  header {
+    overflow: hidden;
+    background-color: #fff;
+    width: 100%;
+    height: 91.6rem;
+  }
+
+  header {
+    position: relative;
+  }
+
+  .logo {
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-left: 2.4rem;
+    margin-top: 5.8rem;
+    background-color: #fff;
+  }
+
+  .see-project-btn a {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    text-decoration: none;
+    color: #396df2;
+    width: 12rem;
+    height: 4rem;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.7rem;
+    line-height: 2.4rem;
+    border-radius: 8px;
+    background-color: #fff;
+    border: 1px solid #396df2;
+    padding-left: 0.8rem;
+  }
+
+  .logo h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1.8rem;
+    line-height: 2rem;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .see-project-btn1 a {
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.7rem;
+    line-height: 2.4rem;
+    color: #fff;
+  }
+
+  .logo h2 a {
+    text-decoration: none;
+    width: 7.6rem;
+    height: 2rem;
+  }
+
+  .see-project-btn a :hover {
+    background-color: #396df2;
+  }
+
+  .logo h2 a :hover {
+    text-decoration: underline;
+  }
+
+  .logo ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    width: 50rem;
+  }
+
+  .logo ul li a {
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.6rem;
+    line-height: 2rem;
+    display: flex;
+    color: #344563;
+    padding: 3.2rem 1rem;
+    margin-right: 2.4rem;
+  }
+
+  .wrapper {
+    max-width: 100%;
+    display: grid;
+    gap: 1.2rem;
+    overflow: hidden;
+    background-color: #e5e5e5;
+    border-bottom-left-radius: 0;
+  }
+
+  .header-wrapper,
+  .about-wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    height: 91.6rem;
+    margin-top: 0;
+    margin-left: 0;
+    background-image: url(/assets/images/header-desktop.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-color: #fff;
+    border-bottom-left-radius: 7rem;
+  }
+
+  .text {
+    overflow: hidden;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+    margin-top: 8rem;
+    margin-left: 10rem;
+  }
+
+  .text h2 {
+    display: flex;
+    width: 92rem;
+    height: 12.2rem;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    margin-left: 2.4rem;
+    margin-right: 2.4rem;
+    margin-top: 10rem;
+    font-size: 4.8rem;
+    line-height: 6rem;
```

Comment:
- [ ] Good job by adding the `@media query` as required by the previous reviewer. So I noticed that you should improve your styling to make sure your is **RESPONSIVE**, it can change the width according to the screen you are using, not every time fixed widths. I suggest you remove all fixed widths both from **Mobile** and **Desktop** so your portfolio will adapt according to the screen you are using. 

- [ ] As pointed out by the previous reviewer, your design doesn't quite match the template design yet, particularly when it comes to fonts, alignment, etc. Take a look at the Figma template again and make sure your project matches the template's margins, padding, background images, fonts, and colors among other things. And center the entire content in of the portfolio as from The Figma design, as from your design most of the content is aligned left.

- [ ] Make sure you improve your styling both from **Mobile** and **Desktop** from the **header** to the **footer** and make it **more responsive** by removing the fixed width as I pointed out in the above comments to stick as much as possible to the Figma design. I know you can do that :muscle: :muscle:

Reply:
I change all fixed font size and width to rem